src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,227 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines annotations on GA4GH genomic sequences It includes two
+types of annotations: continuous and discrete hierarchical.
+
+The discrete hierarchical annotations are derived from the Sequence Ontology
+(SO) and GFF3 work 
+
+   http://www.sequenceontology.org/gff3.shtml
+
+The goal is to be able to store annotations using the GFF3 and SO conceptual
+model, although there is not necessarly a one-to-one mapping in Avro records
+to GFF3 records.
+
+The minimum requirement is to be able to accurately represent the current
+state of the art annotation data and the full SO model.  Feature is the
+core generic record which corresponds to the a GFF3 record.
+*/
+protocol SequenceAnnotations {
+
+  import idl "common.avdl";
+  import idl "ontologies.avdl";
+
+  /**
+  Identifier from a public database
+  */
+  record ExternalIdentifier {
```

Comment:
@diekhans We had actually defined ```accessions``` as an" ARRAY containing other names, IDs, URIs of the object". This would be a more general usage, since it includes both external as well as internal IDs.

In general use "accession" == "unique ID" not necessarily a URI or externally exposed ID. In the metadata schema, the idea was not to overpopulate the attribute space; we had ```ExternalIdentifier``` in an earlier version, but since we definitely need an array for collecting IDs (e.g. representation of imported and/or re-labeled data) I'm in favour for dropping ```ExternalIdentifier``` in favour of just catching these IDs in ```accessions```.